client/client.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+import requests
+import urlparse
+from django.conf import settings as django_settings
+
+
+class Client(object):
+
+    def __init__(self):
+        self._configure()
+
+    def _configure(self):
+        self.endpoint = "http://172.16.16.24:4568"
```

Comment:
Get this value from env file as well.

Reply:
We will get it when this project will be live. We have also added a settings named `NODEBB_URL`
in our `env` files but for now we can't access `localhost` from `docker` so we have just hard coded it.